src/Form/Form.php

```diff
@@ -253,7 +252,6 @@ protected function _execute(array $data)
      * @param string|null $field The field name or null to get data array with
      *   all fields.
      * @return mixed
-     * @since 3.7.0
      */
     public function getData($field = null)
```

Comment:
maybe `getData(?string $field = null)`

Reply:
Good catch. Fixed in 31614341e2ce9ca22ec1fcd44ca3a08751aaf916